package.json

```diff
@@ -32,6 +32,7 @@
     "chart.js": "^3.9.1",
     "class-variance-authority": "^0.4.0",
     "clsx": "^1.2.1",
+    "cors": "^2.8.5",
```

Comment:
I don't think we need cors in the dashboard repo.
Might be better to revert `package.json` and `package-lock.json` back to the original state